.github/workflows/ci.yml

```
@@ -0,0 +1,33 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_GEMFILE: ${{ matrix.gemfile }}
+    strategy:
+      matrix:
+        ruby:
+          - 2.4
+          - 2.5
+          - 2.6
+          - 2.7
+        gemfile:
+          - Gemfile
+          - gemfiles/activesupport-5.0.gemfile
+          - gemfiles/activesupport-5.1.gemfile
+          - gemfiles/activesupport-6.0.gemfile
+    name: Ruby ${{ matrix.ruby }} ${{ matrix.gemfile }}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Ruby ${{ matrix.ruby }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - name: Run tests
+        run: |
+          gem install bundler
+          bundle install --jobs 4 --retry 3
+          bundle exec rake
```

If you use `bundler-cache: true` then this becomes:
```suggestion
          bundle exec rake
```